src/openzaak/components/catalogi/admin/utils.py

```diff
@@ -204,9 +204,6 @@ def import_zaaktype_for_catalogus(
                     for old, new in besluittypen_uuid_mapping.items():
                         data = data.replace(old, str(new.uuid))
```

Comment:
Is it needed now? (and the similar lines above with `ZaakTypeInformatieObjectType` )